.github/workflows/release.yml

```diff
@@ -191,6 +209,14 @@ jobs:
       uses: actions/download-artifact@v3
       with:
         name: azureauth-${{ github.event.inputs.version }}-osx-arm64.tar.gz
+    - name: Download osx-arm64 artifact
+      uses: actions/download-artifact@v3
+      with:
+        name: azureauth-${{ github.event.inputs.version }}-osx-arm64.tar.gz

```

Comment:
This looks like a copy paste error, we already download the `osx-arm64` artifact ;)

Reply:
haha yes. I have more update to this PR actually. I will mark it as a draft.